app/views/exit/index/OfficeOfExitView.scala.html

```diff
@@ -49,7 +49,7 @@
                 heading = messages("exit.index.officeOfExit.heading", countryName),
                 caption = Some(messages("exit.caption"))
             ),
-            hint = Some(messages("exit.index.officeOfExit.hint")),
+            hint = Some(messages("exit.index.officeOfExit.hint", customsOffices.head.name.replace("CUSTOMS OFFICE", ""), customsOffices.head.value)),
```

Comment:
Does this need to be dynamic? It has been hardcoded in other places:
`Enter the office location or code, like Dover or GB000060.`